service/controller/clusterapi/v27/resource/vpccidr/create.go

```diff
@@ -5,16 +5,16 @@ import (
 
 	"github.com/giantswarm/microerror"
 
-	"github.com/giantswarm/aws-operator/service/controller/clusterapi/v27/legacykey"
+	"github.com/giantswarm/aws-operator/service/controller/clusterapi/v27/key"
 )
 
 func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
-	cr, err := legacykey.ToCustomObject(obj)
+	cluster, err := key.ToCluster(obj)
```

Comment:
I am moving the resource to the new key package. Will continue like that in the upcoming PRs.